clang-tools-extra/clang-tidy/daedalean/IncludeOrderCheck.cpp

```diff
@@ -11,6 +11,11 @@
 #include "clang/ASTMatchers/ASTMatchFinder.h"
 #include "clang/Lex/PPCallbacks.h"
 #include "clang/Lex/Preprocessor.h"
+#include "llvm/ADT/None.h"
+#include "llvm/ADT/Optional.h"
+#include "llvm/Support/Debug.h"
+
+#define DEBUG_TYPE "daedalean-include-order"
```

Comment:
What is this define doing?

Reply:
It is needed for `LLVM_DEBUG`, to filter debug messages with `-debug-only=daedalean-include-order`

Reply:
https://llvm.org/docs/ProgrammersManual.html#the-llvm-debug-macro-and-debug-option